File: Java Couse/Java/Day1.java
import java.util.*;
public class Day1{
    public static void main(String[] args) {
        System.out.println("hello world");
        int a=1;
        while (a==1){
            System.out.print("Enter the number : ");
            Scanner number = new Scanner(System.in);
            int no = number.nextInt();
            for (int i=0; i<=20; i++){
                System.out.println(i+"*"+no+"="+i*no);
            }
            System.out.println("If you want to continue, Press '1' or else Press '0' \t");
            int check = number.nextInt();
            if (check == 1){
                System.out.println("Programming is going to running again......");
            }
            else{
                break;
            }
        number.close();
        }

        Scanner input = new Scanner(System.in);
        System.out.print("Enter name 1 : ");
        String name1 = input.nextLine();
        System.out.print("Enter name 2 : ");
        String name2 = input.nextLine();
        System.out.println(name1+" and "+name2+" are friends....");
        input.close();
    }
}

File: Java Couse/Python/Day1.py
print("hello world")
check = 1
while check == 1:
    num = int(input("Enter the number : "))
    for i in range(16):
        print(i,"*",num,"=",i*num)
        i=i+1
    check = int(input("If you want to continue, Press '1' or else Press '0' \n\t"))

name1 = input("Enter the name 1 : ")
name2 = input("Enter the name 2 : ")
print(name1," and ",name2," are friends....")